wordle-web/src/views/WordleView.vue

```diff
@@ -1,37 +1,156 @@
 <template>
-  <h1>Wordle Mind Bender</h1>
+  <v-overlay :model-value="overlay" class="align-center justify-center" persistent>
+    <v-progress-circular color="primary" indeterminate size="64" />
+  </v-overlay>
+  <v-overlay :model-value="namePrompt" class="align-center justify-center" persistent>
+    <v-sheet width="300" class="mx-auto">
+      <v-form @submit.prevent>
+        <v-text-field v-model="userName" label="Username"></v-text-field>
+        <v-btn type="submit" block class="mt-2" @click="setUsername">Submit</v-btn>
+      </v-form>
+    </v-sheet>
+  </v-overlay>
+  <v-btn @click="changeUsername"> {{ userName }} </v-btn>
```

Comment:
I really like how you've done the dialog component here, but it's not perfect, there is a requirement to use local storage for the username

Reply:
now stored locally